mod/gc_elgg_sitemap/start.php

```diff
@@ -0,0 +1,380 @@
+<?php
+
+elgg_register_event_handler('init','system','gc_elgg_sitemap_init');
+
+
+function gc_elgg_sitemap_init() {
+//elgg_register_plugin_hook_handler('register','menu:owner_block','gcforums_owner_block_menu');
```

Comment:
lol i should have refreshed the page... but all the unnecessary commented out code are all removed